### PR TITLE
To add new states support to IPR module, using netcommon HTTPAPI connection plugin

### PR DESCRIPTION
**Pull request by justjais**
Depends-on: https://github.com/ansible/ansible-zuul-jobs/pull/1261
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1262

##### SUMMARY

To add new states support to IPR module, using netcommon HTTPAPI connection plugin

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

deepsec_intrusion_prevention_rules

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

File: tests/integration/targets/deepsec_intrusion_prevention_rules/tasks/redirection.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#   set_fact: test_items="{{ test_cases.files | map(attribute='path') | list }}"
+
+# - name: Run test case (connection=ansible.netcommon.httpapi)
+#   include: '{{ test_case_to_run }}'
+#   vars:
+#     ansible_connection: ansible.netcommon.httpapi
+#   with_items: '{{ test_items }}'
+#   loop_control:
+#     loop_var: test_case_to_run
+#   tags: connection_httpapi
```
Comment (rohitthakur2590) [inline tests/integration/targets/deepsec_intrusion_prevention_rules/tasks/redirection.yaml:18]: commented tasks

File: galaxy.yml [hunk 2]
```diff
@@ -3,12 +3,19 @@ authors:
...
+  "ansible.utils": '>=2.0.0'
 license_file: LICENSE
 name: deepsec
 namespace: trendmicro
 description: Ansible Security Collection for Trend Micro Deep Security
 readme: README.md
+# The URL of the originating SCM repository
 repository: https://github.com/ansible-collections/trendmicro.deepsec
+# The URL to any online docs
+documentation: https://github.com/ansible-collections/trendmicro.deepsec
```
Comment (rohitthakur2590) [inline galaxy.yml:15]: is it intended to provide GitHub repo link or do we have documentation available on docs.ansible.com ?

**Review by rohitthakur2590 — APPROVED** [review]

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by mattclay** [comment]
The current test failures will be fixed by https://github.com/ansible/ansible/pull/76478.

The changes in https://github.com/ansible/ansible/pull/76474 are unrelated.

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**